index.js

```diff
@@ -1,38 +1,80 @@
 const HOME_MENU_BUTTONS_ARRAY = [
     {
-        id: "internet-contents",
-        innerText: "Internet Contents"
+        id: "internetcontents-tab",
+        innerText: "Internet Contents",
+        onclick: () => renderInternetContents()
```

Comment:
Parents are redundant

Reply:
Tried and without parents there is an error in line 5: "Cannot access 'renderInternetContents' before initialization"

Reply:
Ok, so do you know what you should do to resolve this error?

Reply:
Yes, solved.